sdk/storage/azure-storage-blobs/CHANGELOG.md

```diff
@@ -4,6 +4,12 @@
 
 ### Features Added
 
+- Bumped up API version to `2020-10-02`.
+- Added new API: `BlockBlobClient::Query()`.
+- Added `ContinuationToken` and `PageSizeHint` in `GetPageRangesOptions`.
+- Added support for listing system containers.
+- Added support for listing deleted root blob with active versions.
+
```

Comment:
Is source auth with OAuth for Copy Blob API supported? I only see the change in the protocol layer.